backend/tests/test_deps.py

```
@@ -13,6 +13,11 @@ def is_configured(self) -> bool:
         return False
 
 
+class _UnavailableFirestore(_UnconfiguredFirebase):
+    def firestore_client(self):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `firestore_client`

Add return type annotation: `None`

(ANN202)